cloud-chatops/INSTALL.md

```diff
@@ -1,12 +1,72 @@
 # Deploying Cloud ChatOps
 This document covers the following: creating a Slack Workspace and Application, finding / generating Slack and GitHub tokens and deploying the application onto a host. 
 
+> **_NOTE:_** The installation has only been tested on Ubuntu (20.04) Linux Distributions and may need to be modified to work on others such as Rocky Linux.
+
 ## Contents:
+[Quick Start](#recommended-installation-quick-start)<br>
 [Slack Configuration](#slack-configuration)<br>
 [Slack Tokens](#slack-tokens)<br>
 [Deployment Configuration](#deployment-configuration)<br>
 [Deployment](#deployment)<br>
 
+### Recommended Installation (Quick Start):
+
+Below are instructions for the recommended installation.
+This uses docker compose and an auto updating script to keep the application on the latest version.<br>
+
+#### Prerequisites:
+
+- Set up Slack Workspace and Application [here](#slack-configuration)
+- Retrieved Slack bot user and app tokens[here](#slack-tokens)
+- Generated a GitHub personal access token
+- Installed Docker and Docker Compose [here](https://docs.docker.com/engine/install/ubuntu/)
+
+#### Installation:
+
+1. Make a directory in `/etc` to store all ChatOps related files. Set group ownership to docker so other users (update script) can access the files.
+    ```shell
+    # Create directories
+    sudo mkdir -p /etc/chatops/secrets
+    
+    # Set permissions
+    sudo chown -R $USER:docker /etc/chatops
+    sudo chmod 774 /etc/chatops
+    sudo chmod 770 /etc/chatops/secrets
+   
+    # Clone repository into directory
+    git clone https://github.com/stfc/cloud-docker-images.git /etc/chatops/cloud-docker-images
+    ```
+2. Edit the `template_config.yml` and `template_secrets.json` to create your respective config and secrets file. See [here](#deployment-configuration) for more detail.
+   ```shell
+   # Copy files into secrets folder
+   cp <path_to_file>./template_config.yml /etc/chatops/secrets/config.yml
+   cp <path_to_file>./template_secrets.json /etc/chatops/secrets/secrets.json
+   
+   # If you edited the template_secrets.json from the cloned repository in /etc/chatops
+   # You should delete / reset the file as the permissions are too open
+   
+   git reset --hard origin/master 
+   # Or
+   rm <path_to_file>/template_secrets.json
+   ```
+3. Need to add the Ubuntu user to the docker group for the cron script:
+   ```shell
+   sudo usermod -aG docker ubuntu
+   ```
+4. Move Cron script and start container:
+   ```shell
+   # Copy script to etc/cron.< hourly | daily | monthly >
+   sudo cp /etc/chatops/cloud-docker-images/cloud-chatops/chatopscron /etc/cron.daily/
+
+   # Test the script works and launch the container
```

Comment:
```suggestion
   # You can test that the script is running correctly by using this command. The --test flag says it won't run the script but it will test if it has been added to the directory correctly
  run-parts --test /etc/cron.<timeframe>
  
  #if you don't receive any output, the script will not run.
  
```

Reply:
Feel free to reword that first comment ` # You can test that...` if you think the suggestion is valid, seems a bit too wordy but I couldn't think of a better way of saying it.

Reply:
Changes made.